packages/lib-classifier/src/plugins/tasks/DrawingTask/models/DrawingTask.js

```diff
@@ -8,7 +8,7 @@ const toolModels = Object.values(tools)
 
 const Drawing = types.model('Drawing', {
   activeToolIndex: types.optional(types.number, 0),
-  annotation: types.maybe(types.safeReference(DrawingAnnotation)),
+  annotation: types.safeReference(DrawingAnnotation),
```

Comment:
Does `types.safeReference` allow undefined values? The task annotation will be undefined until one is explicitly passed to the task from the classification.

Reply:
Note to myself: I have been using `type.safeReference` completely wrong throughout the classifier.

Reply:
Yes, it does. Wrapping it in `types.maybe` was redundant.

Reply:
I've been using `safeReference` completely wrong, so I'll open a PR to fix that.